### DISCUSSION TITLE
About compression format

### DISCUSSION THREAD

**Discussion by jueqing1015** [dlemstra/Magick.NET#677] [General]
@dlemstra 
I need to use YUV4:4:4 now, how do I set it up?

**Comment by dlemstra** [comment]
Are you talking about using `4:4:4` Chrome Subsampling? You can set the `JpegSamplingFactor` with the `JpegWriteDefines` and pass that to the `Write` method.

**Reply by jueqing1015** [reply]
Ok, I don’t know if the 4:4:4 Chrome Subsampling you are referring to is the same as what I’m talking about. I’m talking about JPEG YUV4:4:4. If it’s the same, how can I use it in MAGICK.NET. I didn't find how to use it. Thank you

**Reply by dlemstra** [reply]
It this what you are looking for:

```C#
using (var image = new MagickImage("yourImage"))
{
    image.Write("output", new JpegWriteDefines
    {
        SamplingFactor = JpegSamplingFactor.Ratio444
    });
}
```
